_docs_use-cases/use-case-transport-create-relationship-with-relationshiptemplate.md

```diff
@@ -42,12 +42,17 @@ api_route_regex: ^POST /api/v2/Relationships$
+This use case intends to create a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) based on a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), which was previously received.
+The Relationship will be established with the RelationshipTemplate's creator.
+This use case must always be applied if the `content` of the RelationshipTemplate is an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent).
+However, if it is a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent), it usually does not have to be executed manually anymore.
+Instead, the [Request Module]({% link _docs_explore/61-runtime.md %}#request-module), which is enabled by default, takes care of this automatically.
+For information on how to [establish a Relationship]({% link _docs_integrate/establish-relationships.md %}) based on a [RelationshipTemplate with RelationshipTemplateContent]({% link _docs_integrate/establish-relationships.md %}#relationshiptemplate-with-relationshiptemplatecontent) and an enabled Request Module, refer to the corresponding scenario documentation.
 
 ## Parameters
 
-- `templateId` references the RelationshipTemplate that was received from a party.
-- `creationContent` is an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent) and will be stored inside the Relationship. It can be analyzed by the RelationshipTemplate's creator to decide whether to accept the Relationship.
+- `templateId` references the RelationshipTemplate that was received from its creator.
+- `creationContent` is usually an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent) when the RelationshipTemplate's `content` is an ArbitraryRelationshipTemplateContent and the use case is applied manually and a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) when the RelationshipTemplate's `content` is a RelationshipTemplateContent and the [Request Module]({% link _docs_explore/61-runtime.md %}#request-module) automatically takes care of the execution of the use case. It will be stored inside the Relationship and can be analyzed by the creator of the RelationshipTemplate to decide whether to [accept the Relationship]({% link _docs_use-cases/use-case-transport-accept-relationship.md %}).
```

Comment:
Please split up this sentence, I am having trouble understanding it with all the "and"s. Also, use "if"s instead of "when"s.